src/tiledb/cloud/vcf/ingestion.py

```diff
@@ -707,14 +739,14 @@ def file_size(uri: str) -> int:
                     # Check for index issues
                     index_uri = find_index(vcf_uri)
                     if not index_uri:
-                        status = "" if status == "ok" else status + ","
-                        status += "missing index"
+                        status = "" if status == Status.READY else status + ","
+                        status += Status.MISSING_INDEX
```

Comment:
Seeing this, I wonder if status shouldn't be a stack (list) instead of a string of comma-separated values.

Reply:
Yeah, I don't think string is quite right here. I would like to revisit the whole manifest at some point but this PR is just a bug fix!